### PR TITLE
modify ci file name

### PR DESCRIPTION
**Pull request by chunyu3**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

-  rename the ci.yml file of template under “sdk/template/.content/testResource/ci.yml” to avoid pipeline tool creating pipeline for it. issue: https://github.com/Azure/azure-sdk-for-net/issues/29123

### CODE REVIEW

File: sdk/template/.template.config/template.json [hunk 1]
```diff
@@ -438,6 +436,14 @@
...
         ]
       }
+    },
+    "ciFileName": {
+      "type": "parameter",
+      "datatype":"text",
+      "isRequired": false,
+      "defaultValue": "ci",
+      "description": "ci file name.",
+      "fileRename": "ci-template"
```
Comment (weshaggard) [inline sdk/template/.template.config/template.json:446]: How does this map to the file? Does it ignore the file extension?

**Review by weshaggard — APPROVED** [review]
One question but overall looks reasonable assuming the rename doesn't need the extension.

File: sdk/template/.template.config/template.json [hunk 2]
```diff
@@ -438,6 +436,14 @@
           }
         ]
       }
+    },
+    "ciFileName": {
```
Comment (heaths) [inline sdk/template/.template.config/template.json:440]: Why does it need to be a parameter? This just complicates using it when the target file name is known. Just use a rename directive: https://github.com/dotnet/templating/wiki/Reference-for-template.json#source-definition

This is a good reference to see other capabilities of templates. The fewer parameters the template has, the easier it is to use.

**Review by heaths — CHANGES_REQUESTED** [review]

File: sdk/template/.template.config/template.json (hunk 2 above)
Comment (chunyu3) [reply sdk/template/.template.config/template.json]: Thanks @heaths. I updated to use 'rename directive' to rename the file name. Please help to review again. Thanks,

**Review by heaths — APPROVED** [review]

File: sdk/template/.template.config/template.json (hunk 1 above)
Comment (chunyu3) [reply sdk/template/.template.config/template.json]: No, it is just replace 'çi-template' to 'ci' for the file name.
We have changed to use 'rename directive' to do this.

**Comment by chunyu3** [comment]
Fix https://github.com/Azure/azure-sdk-for-net/issues/29123

**Pull request merged**